### ISSUE TITLE
Regular text with insufficient contrast.

### ISSUE THREAD

**Issue by ErickOdaga** [ErickOdaga/CODA-Website-A11Y-Homepage#1]
The "read more" text does not have a sufficient contrast ratio of 4.5:1.
Foreground: #E7AD59
Background: #F0F0F0
Contrast ratio: 1.8:1

![image](https://github.com/ErickOdaga/CODA-Website-A11Y-Homepage/assets/71803406/cc7b69c3-c9e9-47ff-9dd2-555c71352462)
